mappings/net/minecraft/data/client/ItemModelGenerator.mapping

```diff
@@ -23,9 +23,10 @@ CLASS net/minecraft/unmapped/C_cfvrpkog net/minecraft/data/client/ItemModelGener
 	METHOD m_llpeiyoc createArmorJson (Lnet/minecraft/unmapped/C_ncpywfca;Ljava/util/Map;Lnet/minecraft/unmapped/C_ncpywfca;)Lcom/google/gson/JsonObject;
 		ARG 1 id
 		ARG 2 textures
-		ARG 3 overrideArmorMaterialId
+		ARG 3 modelId
```

Comment:
I don't think this is correct.

It's used here:
```java
jsonObject2.addProperty("model", this.suffixTrim(id, lv.getArmorMaterialOverride(modelId)).toString());
```
where `lv.getArmorMaterialOverride(modelId)).toString()` is passed to `suffixTrim`'s `materialName` param

```suggestion
		ARG 3 materialOverrideId
```

Reply:
ok i get what happened here
i got it right the first time, this is what i meant, but i then proceeded to misunderstand the code when reevaluating the name. embarrassing for me